### ISSUE TITLE
Bug(subtitle): vietnamese underdot tone mark is rendered incorrectly

### ISSUE THREAD

**Issue by boydaihungst** [jarnedemeulemeester/findroid#691]
**Describe the bug**
External Vietnamese subtitle is rendered "underdot" tone mark incorrectly. Please check images below.
Underdot tone mark in Vietnamese:
https://vietnamesetypography.com/tone-marks/#underdot

**To Reproduce**
1. Enable mpv player.
2. Go to any movie or tv shows 
3. Click 'play' button.
4. Wait until external subtitles is downloaded
5. Check any word in the subtitles which includes Vietnamese 'underdot' tone mark. 

**Expected behavior**
Underdot tone mark should display under vowels: ạ, ẹ, ị, ọ, ụ, and ỵ
This bug can be fixed by changing the default mpv font to google sans, open sans or noto sans in android > data... But it's not persistent, after updating app the font will restore to default.

**Screenshots**
Incorrectly display in findroid ("phục" and "lạc" words):
![Screenshot_20240315-131947_Findroid](https://github.com/jarnedemeulemeester/findroid/assets/38396158/bb8410af-76b8-4617-a247-06fa5acde263)

Correctly display in jellyfin official app:
![Screenshot_20240315-131933_Jellyfin](https://github.com/jarnedemeulemeester/findroid/assets/38396158/67ce191f-6cef-4eb4-b823-ee64e4cbe140)

**Device info (please complete the following information):**
 - Device: Red magic 6s pro
 - Android version: 14
 - App version: 0.14.1
 - Jellyfin version: 10.8.13

**Additional context**
Subtitles at line 74:
[Beast Tamer - S01E01.zip](https://github.com/jarnedemeulemeester/findroid/files/14611700/Beast.Tamer.-.S01E01.zip)

**Comment by Natanel-Shitrit** [comment]
The official app uses ExoPlayer, which uses system-wide subtitles settings for rendering - which means it doesn't rely on the source file format.

This is not a Findroid problem, it could be a problem with the subtitles format or rather how mpv-android is rendering it.
I would recommend opening an issue in the [mpv-android issue tracker](https://github.com/mpv-android/mpv-android/issues), they will probably help you better with these type of issues.

**Comment by boydaihungst** [comment]
Yeah you are right. I tested in mpv android. Mpv doesn't use android caption font at the moment, it also can't parse android system font. So it will fallback to a subfont.tff (which I can replace with other font to fix the problem, as I mentioned above). #216 in this issue, look like he fixed the problem by changing the font at building stage.

https://github.com/mpv-android/mpv-android/issues/96#issuecomment-364439797

https://github.com/mpv-android/mpv-android/issues/748.

**Comment by Natanel-Shitrit** [comment]
@boydaihungst 
Gotcha, thanks for the thorough explanation.
Since it's not exactly a bug (and just a side effect - font not being compatible with some languages)
I would recommend opening a feature request for customizing the subtitles font for mpv. (since it's already supported for ExoPlayer)
